modules/videostab/src/stabilizer.cpp

```diff
@@ -391,6 +391,16 @@ void TwoPassStabilizer::runPrePassIfNecessary()
         {
             if (frameCount_ > 0)
             {
+                Mat feature_mask = Mat();
+                if (maskSource_) {
+                    Mat mask_frame = maskSource_->nextFrame();
+                    if (!mask_frame.empty())
+                    {
+                        feature_mask = Mat(mask_frame.size(), CV_8UC1);
```

Comment:
Initializer here is not needed too. `dst` can be allocated by processing functions.